Packs/CVSS/CVSSCalculator/CVSSCalculator.py

```diff
@@ -0,0 +1,333 @@
+import math
```

Comment:
Move all the CVSSCalculator files under `Packs/CVSS/Script/CVSSCalculator`

Reply:
Done.